Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -93,7 +142,15 @@ async function executeMSDeploy(msDeployCmdArgs) {
     });
 
     try {
-        await tl.exec("msdeploy", msDeployCmdArgs, <any>{failOnStdErr: true, errStream: errObj});
+        tl.debug("the argument string is:");
+        tl.debug(msDeployCmdArgs);
```

Comment:
Username and password is masked [here](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts#L163-L164).


@arjgupta, is it better to move the masking logic inside MSDpeloyCmgArgs method ?
This way, we can be sure that the credentials are masked wihtout referring to the main file.